requirements.in

```diff
@@ -1,17 +1,16 @@
-django==3.2.25
-requests==2.31.0
-requests_oauthlib
+django==4.2.20
```

Comment:
Let's bump to Django 5.1 if you don't see any issues with that so this project is in position for the 5.2 LTS release. But only if there aren't any complications, otherwise fine with leaving it as 4.2 for now.